R/step-setnames.R

```diff
@@ -0,0 +1,37 @@
+step_setnames <- function(x, ..., in_place, rename_groups = FALSE) {
+  stopifnot(is_step(x))
+  stopifnot(is_bool(in_place))
+  stopifnot(is_bool(rename_groups))
+
+  sim_data <- simulate_vars(x)
+  locs <- tidyselect::eval_rename(expr(c(...)), sim_data)
```

Comment:
This logic wasn't previously applied in (e.g.) `pivot_wider.dtplyr_step()`. Is it likely to cause problems?